modules/form-field/react/lib/Hint.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
+import {styled, Themeable} from '@workday/canvas-kit-labs-react-core';
 import {ErrorType} from '@workday/canvas-kit-react-common';
 import {spacing, type} from '@workday/canvas-kit-react-core';
 
-export interface HintProps extends React.HTMLAttributes<HTMLParagraphElement> {
+export interface HintProps extends Themeable, React.HTMLAttributes<HTMLParagraphElement> {
```

Comment:
`Label` isn't flipping to the right side

Reply:
It won't flip if your text is in english but if you use a RTL language it should. The `Error:` and `Alert:` is not translatable at this point, I created an issue: https://github.com/Workday/canvas-kit/issues/385

![image](https://user-images.githubusercontent.com/723109/71482745-7cfa9a80-2804-11ea-8928-b47eb74039c8.png)